bridge/client/app/evaluation-board/evaluation-board.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container" *ngIf="error">
+  <dt-empty-state>
+    <dt-empty-state-item>
+      <dt-empty-state-item-img>
+        <img class="mt-2" src="/assets/keptn_logo.png" />
+      </dt-empty-state-item-img>
+      <ng-container [ngSwitch]="error">
+        <ng-container *ngSwitchCase="'contextError'">
+          <dt-empty-state-item-title class="mt-2" aria-level="2">Traces for <span class="italic" [textContent]="contextId"></span> not found</dt-empty-state-item-title>
+          <p>Sorry, we couldn't load any traces with that shkeptncontext. Check our <a href="https://keptn.sh/docs/quickstart/" target="_blank">Quick Start</a> instructions on how to <a href="https://keptn.sh/docs/0.6.0/manage/project/" target="_blank">setup a project</a> or go back to <a [routerLink]="['/dashboard']">your dashboard</a>.</p>
+        </ng-container>
+        <ng-container *ngSwitchDefault>
+          <dt-empty-state-item-title class="mt-2" aria-level="2">Some error occured</dt-empty-state-item-title>
+          <p>Sorry, some error occured. Check our <a href="https://keptn.sh/docs/quickstart/" target="_blank">Quick Start</a> instructions on how to <a href="https://keptn.sh/docs/0.6.0/manage/project/" target="_blank">setup a project</a> or go back to <a [routerLink]="['/dashboard']">your dashboard</a>.</p>
+        </ng-container>
+      </ng-container>
+    </dt-empty-state-item>
+  </dt-empty-state>
+</div>
+<div class="container" *ngIf="!error && !evaluations">
+  <div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="15px">
+    <dt-loading-spinner></dt-loading-spinner>
+    <p>Loading ...</p>
+  </div>
```

Comment:
This probably could be useful as reusable component for other screens as well 🤔

Reply:
Yes, since the same template is also used in the `project-board`. There is already an issue to split the `project-board` into smaller components: #1989